### ISSUE TITLE
multinode problem

### ISSUE THREAD

**Issue by jms0214** [EOSIO/eos#6739]
Hello. I tried test about this site. "https://developers.eos.io/eosio-nodeos/docs/local-multi-node-testnet"
When I execute second node like this,
$ nodeos --producer-name lucy --plugin eosio::chain_api_plugin --plugin eosio::net_api_plugin --http-server-address 0.0.0.0:8889 --p2p-listen-endpoint 0.0.0.0:9877 --p2p-peer-address 10.30.113.44:9876 --config-dir node2 --data-dir node2 --private-key [\"EOS8MuVeK85cqampVfVvjTuDuWiEXaQH4iJrLNyWJfLVfESvcHxWs\",\"5JXdgiQf4kiHxw8rUUHasJGcpdhbi7XgJTAGBg4NAkAbjRcKWuC\"] --hard-replay-blockchain

error occur like this
info  2019-02-14T07:32:04.710 thread-0  chain_plugin.cpp:335          plugin_initialize    ] initializing chain plugin
info  2019-02-14T07:32:04.710 thread-0  chain_plugin.cpp:494          plugin_initialize    ] Hard replay requested: deleting state database
info  2019-02-14T07:32:04.710 thread-0  block_log.cpp:368             repair_log           ] Recovering Block Log...
info  2019-02-14T07:32:04.710 thread-0  block_log.cpp:388             repair_log           ] Moved existing blocks directory to backup location: '/home/lucy/node2/blocks-2019-02-14T07:32:04.710'
info  2019-02-14T07:32:04.710 thread-0  block_log.cpp:393             repair_log           ] Reconstructing '/home/lucy/node2/blocks/blocks.log' from backed up block log
warn  2019-02-14T07:32:04.711 thread-0  chain_plugin.cpp:707          plugin_initialize    ] 3190000 block_log_exception: Block log exception
Block log was not setup properly
    {}
    thread-0  block_log.cpp:407 repair_log
error 2019-02-14T07:32:04.711 thread-0  main.cpp:129                  main                 ] 3190000 block_log_exception: Block log exception
Block log was not setup properly
    {}
    thread-0  block_log.cpp:407 repair_log
rethrow
    {}
    thread-0  chain_plugin.cpp:707 plugin_initialize

Why this problem happens?

**Comment by jgiszczak** [comment]
Your old blocks log is completely invalid.  Simply start up without the `--hard-replay-blockchain` option.  You may also delete `home/lucy/node2/blocks-2019-02-14T07:32:04.710`.

**Comment by jms0214** [comment]
I deleted that folder. And I executed without `--hard-replay-blockchain`. But error still occur like this.

info  2019-02-15T08:46:55.669 thread-0  chain_plugin.cpp:335          plugin_initialize    ] initializing chain plugin
warn  2019-02-15T08:46:55.669 thread-0  chain_plugin.cpp:707          plugin_initialize    ] 3190000 block_log_exception: Block log exception
Block log was not setup properly.
    {}
    thread-0  block_log.cpp:537 extract_genesis_state
error 2019-02-15T08:46:55.669 thread-0  main.cpp:129                  main                 ] 3190000 block_log_exception: Block log exception
Block log was not setup properly.
    {}
    thread-0  block_log.cpp:537 extract_genesis_state
rethrow
    {}
    thread-0  chain_plugin.cpp:707 plugin_initialize
